scala-package/mxnet-demo/java-demo/README.md

```diff
@@ -42,14 +42,14 @@ The `SCALA_PKG_PROFILE` should be chosen from `osx-x86_64-cpu`, `linux-x86_64-cp
 
 
 ## Run
```

Comment:
Please fix the first line of the sample's description.
`This is an project created to use Maven-published Scala/Java package with two Java examples.` --> 
```
This is a project demonstrating how to use the Maven published Scala/Java MXNet package. 
The examples provided include:
* NDArray creation
* Object detection using the Inference API
```
Removing the specific number makes it easier to add more examples later, plus listing them tells the reader what to expect without having to read through the whole doc.

Reply:
Also, in the setup section, please define what folder the user is supposed to run the `mvn package` command. I assume it is from this same `java-demo` folder, but please specify.